src/button/button.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * JavaScript 'shim' to trigger the click event of element(s) when the space key is pressed.
```

Comment:
Comments in this file are generally 💯. Is the doc block style new? Are there new conventions we're establishing that need to be documented?

Reply:
We started using them on the lib jest helpers, seems like a good thing to try to do everywhere it makes sense.